IPython/html/static/base/js/utils.js

```diff
@@ -39,6 +39,20 @@ define([
         );
     };
 
+    /**
+     * Wait for a config section to load, and then load the extensions specified
+     * in a 'load_extensions' key inside it.
+     */
+    function load_extensions_from_config(section) {
+        section.loaded.then(function() {
+            if (section.data.load_extensions) {
+                var nbextension_paths = Object.getOwnPropertyNames(
+                                            section.data.load_extensions);
+                IPython.load_extensions.apply(this, nbextension_paths);
```

Comment:
-1 on using IPython global, as load_extension is defined in same module. 

if you fix it could you remove as the same time `IPython.load_extension` and replace it with a `var load_extension`at the top of this file, export it and bind it to IPython later?